### ISSUE TITLE
interference with osx basic Shortcuts

### ISSUE THREAD

**Issue by JeromeNamNam** [jigish/slate#195]
Hi guys,
I find out with my Slate installation, that when slate is running my favourite finder shortcut doesn't work anymore.

Command-Down Arrow - Open highlighted item
Command-Up Arrow - Open the folder that contains the current folder

Is it because of something in the config file? or the soft in it self?

nb: in place of opening the folder it switches between windows.

Any idea?

regards,

**Comment by loshlee** [comment]
If you aren't using a .slate file, those shortcuts appear to conflict with the focus down and focus up items in the Slate default configuration. https://github.com/jigish/slate/blob/master/Slate/default.slate

If you are using a .slate file, you may want to change any occurrence of "bind down:cmd" and "bind up:cmd".

Laine

**Comment by JeromeNamNam** [comment]
Yes I'm using a .slate config file.
and by commenting
#bind up:cmd       focus up
#bind down:cmd     focus down
problem is solved.
Thank you a lot !

**Comment by phmongeau** [comment]
Considering it's not the first time someone complains about the default
keybindings, maybe they should be changed to something that doesn't
conflict with the osx shortcuts?

Philippe Mongeau

**Comment by JeromeNamNam** [comment]
Yes,
And/Or give some explanations about how to set or modify it.

Regards,

**Comment by josh-** [comment]
I was thinking about submitting a pull request that, upon first run, would give you the option to copy the `default.slate` to  `~/.slate` and then open that in your default text editor so you can edit it. Currently this process isn't hugely user friendly, and can only really be understood by reading the README.

What do you think @JeromeNamNam and @phmongeau?

**Comment by JeromeNamNam** [comment]
Yes It's a good idea, but I think the readme also has to define clearly which keyboard key is:
#bind 
because I really didn't though it was the apple one.

Kind regards,

**Comment by klokie** [comment]
Also, defaults of right:cmd and left:cmd clash with standard web browser forward and backward controls, and right:alt and left:alt clash with standard OS X text controls (next/previous word). What would be great would be to adjust these defaults so as not to clash with de facto standard keys, and pop up a dialog upon first use that allows the user to edit them; alternatively add a menu item to the Slate menu that will open the config file in a text editor (as not all users will know where to find the .slate defaults file). Thanks!

**Comment by ianjevans** [comment]
+1 to the above comment by klokie. It was extremely frustrating to not have keyboard shortcuts of option-left/right when using a web browser, and the documentation makes no mention of this. I'm surprised that the default keybindings in Slate override the default keybindings in OS X.

**Comment by mhartl** [comment]
I spent half an hour in a semi-panic about the lost keybindings (which I use hundreds of times an hour) before I realized Slate had broken Cmd-<arrow key>. I strongly recommend changing the current defaults.

**Comment by danielpops** [comment]
> [mhartl] I spend half an hour in a semi-panic

+1. The [still] current defaults hijack too many common keystrokes, such as cmd+left/right, option+left/right, etc...  I also spent a half-hour panic before I binary searched and attributed the strange key behavior to the default settings.  plz change these settings and/or expose a UI option that shows the overridden keyboard shortcuts

**Comment by mattr-** [comment]
What would be used for the defaults if we changed them?

**Comment by mattr-** [comment]
I envision finding a set of shortcuts that don't conflict with the common OS X keyboard shortcuts to be difficult. Is there a set of shortcuts that we should absolutely avoid?

**Comment by danielpops** [comment]
I'm honestly not sure what would be the best set.  Maybe default to no
shortcuts at all?  I personally don't use slate for most of it's purposes.
I only use it to remember the snapshot of all my windows locations sizes
etc... mainly for docking/undocking the macbook.  Maybe I'm not the right
person to ask :)

Daniel Popescu
818-625-0823

**Comment by jneubrand** [comment]
I'd like to suggest keeping the defaults as they are, but additionally showing a dialog on the first launch—something that informs users how to configure slate…
